src/reynir_correct/errtokenizer.py

```diff
@@ -1789,6 +1847,65 @@ def suggest_word(
         token.add_corrected_meanings(m)
         return token
 
+    def add_ritmyndir_error(token: CorrectToken) -> CorrectToken:
+        """Add an error with corresponding correct value and details given info in Ritmyndir"""
+        # TODO At the moment the code assumes only one correct value is available in data
+        code = Ritmyndir.get_code(token.txt)
+        if code == "R001":
```

Comment:
This would be slightly more efficient with a predefined constant frozenset() at the beginning of the file, with the neutral codes in it, then a single membership test here would suffice instead of three IFs. I.e.: `NEUTRAL_RITMYNDIR_CODES = frozenset((...))`  and `if code in NEUTRAL_RITMYNDIR_CODES:`

Reply:
Done, kept the original comments for each type in the frozenset.